src/python_pachyderm/experimental/mixin/pps.py

```diff
@@ -82,6 +88,8 @@ def list_job(
         history: int = 0,
         details: bool = False,
         jqFilter: str = None,
+        project_name: str = None,
+        project_filter: List[str] = None,
```

Comment:
Missing docstring and usage of `project_filter`

Reply:
Yeah the "projects_filter" (named "projects" in the proto messages) are a little un-intuitive in my opinion. Any projects that are listed under the "project_filter" are filtered _out_ of the response.

Moreover, having both "project_name" and "project_filter" is really not ideal. We should just be having users pass "Repo" objects that contain the information like project, repo, branch, etc. all bundled together. But that would be a huge change and we are trying to make this as backwards-compatible as possible.